OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -493,7 +498,13 @@ public int MovementSpeed
 			get { return Util.ApplyPercentageModifiers(Info.Speed, speedModifiers); }
 		}
 
-		public Pair<CPos, SubCell>[] OccupiedCells() { return NoCells; }
+		public Pair<CPos, SubCell>[] OccupiedCells()
+		{
+			if (!self.IsAtGroundLevel())
```

Comment:
`OccupyCells` can be called by arbitrary things at arbitrary times, so we should still be returning `NoCells` during normal flight to avoid bogosity.

Reply:
Okay. Tied this to `RemoveInfluence` to mirror `AddInfluence`.